syntax = "proto3";

package vuelos;

message Request {
  string ip_client = 1;
  int32 altura = 2;
}

message Response {
  bool permiso = 1;
  int32 pista_at = 2;
  int32 altura = 3;
  string ip_server = 4;
  string aero = 5;
}

service Asig {
  rpc Asig_pistas (Request) returns (Response);
  rpc Asig_salida (Request) returns (Response);
  rpc Asig_chequeo (Request) returns (Response);
}
